library/include/borealis/list.hpp

```diff
@@ -162,9 +163,10 @@ class InputListItem : public ListItem
   protected:
     std::string helpText;
     int maxInputLength;
-
+    enum keyboard_disable_bitmask kbdDisableBitmask;
```

Comment:
Does it make sense to allow changing the bitmask for _all_ list items, rather than only for free input ones?

Reply:
I modified the InputListItem. did I missed something ?
```
// A list item which spawns the swkbd
// to input its value (string)
class InputListItem : public ListItem
{
  protected:
    std::string helpText;
    int maxInputLength;
    enum keyboard_disable_bitmask kbdDisableBitmask;

```